serial_node:
  ros__parameters:
    port_name: "/dev/ttyUSB0"
    baud_rate: 9600

